frontend/src/scenes/experiments/ExperimentView/Goal.tsx

```diff
@@ -113,7 +115,10 @@ export function ExperimentGoalModal({ experimentId }: { experimentId: Experiment
                     </LemonButton>
                     <LemonButton
                         disabledReason={
-                            goalInsightDataLoading && 'The insight needs to be loaded before saving the goal'
+                            (goalInsightDataLoading && 'The insight needs to be loaded before saving the goal.') ||
```

Comment:
@jurajmajerik Periods at the end of tooltips, right?

Reply:
I'm not sure what the convention here should be, fine by me 👍 

Not mentioned in our style guide: https://posthog.com/handbook/growth/marketing/posthog-style-guide